floor/floor/processor/electricity.py

```diff
@@ -83,9 +87,9 @@ def __init__(self, point1, point2):
 
         # The points translated to x & y
         self.p1x = point1 % 8
-        self.p1y = int(point1 / 8)
+        self.p1y = int(old_div(point1, 8))
```

Comment:
With the int (looks like I thought this would be a float) we can keep the '/' operator rather than use old_div.